.github/workflows/int-and-connected-test-run.yml

```diff
@@ -229,6 +229,7 @@ jobs:
     name: "Save execution reports and notify"
     timeout-minutes: 60
     needs:
+      - test_check
```

Comment:
Just curious - why does this fix the false negative?

Reply:
Ah, this was annoying.  I used the `needs` value when figuring out job statuses and you need to explicitly add the values that you need (I assumed incorrectly that it would pull it in transitively from the connected and integration tasks)

Reply:
> I assumed incorrectly that it would pull it in transitively from the connected and integration tasks

This should really be an error. How can you reference a `needs` item in a job and not have it in your `needs` clause?

Reply:
100%...I'm surprised I didn't get an NPE.  Actions are cool and all but the dev experience for them is pretty bad